### DISCUSSION TITLE
Alt+Enter While map is loading caused crash and broke install

### DISCUSSION THREAD

**Discussion by hburn7** [ppy/osu#21972] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

I launched osu! and loaded a map to be played with the Auto mod without doing anything else (this was my only reason for opening osu!). I pressed Alt+Enter to change the screen resolution out of exclusive full screen while the map was loading (the in-between period where the osu! logo is center-screen, after map is clicked in selection, before notes appear). osu! crashed, and I cannot launch osu! anymore. The video attached is what happens when I try to launch osu! now. This behavior persists after restarting. If you look very closely at the video, you can see the opening animation does start, but then crashes.

### Screenshots or videos

https://user-images.githubusercontent.com/38370573/210149880-88453d40-d5e9-4d9a-aaa2-88c05ccf8f9f.mp4

### Version

2022.1228.0.0

### Logs

[input.log](https://github.com/ppy/osu/files/10327647/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10327648/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10327649/network.log)
[performance.log](https://github.com/ppy/osu/files/10327650/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10327651/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10327652/updater.log)
[database.log](https://github.com/ppy/osu/files/10327653/database.log)

**Comment by bdach** [comment, marked as answer]
Please try the following:

- Updating graphics card drivers
- Disabling shadowplay in nvidia settings

and seeing if any of those help.

**Reply by hburn7** [reply]
Yep, turning this off solved this, sorry about that.
![image](https://user-images.githubusercontent.com/38370573/210150164-53bb65ff-59fa-4f6f-8256-c40df9b82752.png)
